lib/src/signed_video_h26x_auth.c

```diff
@@ -613,7 +667,7 @@ verify_hashes_without_sei(signed_video_t *self)
       item = item->next;
       continue;
     }
-
+    item->used_in_gop_hash = true;
```

Comment:
Is it really necessary to mark the items if you remove the mark as soon as you leave the while loop?